src/chains/ethereum/ethereum/src/blockchain.ts

```diff
@@ -623,11 +612,12 @@ export default class Blockchain extends Emittery<BlockchainTypedEvents> {
 
   createVmFromStateTrie = async (
     stateTrie: GanacheTrie | ForkTrie,
-    allowUnlimitedContractSize: boolean,
```

Comment:
I didn't see any value in having this parameter be passed in vs just reading from the settings. In all non-test uses of this function, it was pulling this flag from `options.chain.allowUnlimitedContractSize` anyways, so why not just set it within? This allowed the initialization of the `SimulationHandler` class to be just a bit cleaner

Reply:
Any further thoughts on how we should handle this? Should we add the parameter back?

Reply:
It's fine the way you have it.